nx.json

```diff
@@ -299,28 +195,89 @@
         "{projectRoot}/dist/middlewares/**/*.d.ts",
         "{projectRoot}/dist/middlewares/**/*.d.ts.map",
         "{projectRoot}/build/.tsbuildinfo.builders"
-      ]
+      ],
+      "cache": true
     },
     "build-plugins": {
-      "dependsOn": [
-        "^build",
-        "compile"
-      ],
-      "inputs": [
-        "plugins"
-      ],
-      "outputs": [
-        "{projectRoot}/dist/plugins",
-        "{projectRoot}/build/plugins"
-      ]
+      "dependsOn": ["^build", "compile"],
+      "inputs": ["plugins"],
+      "outputs": ["{projectRoot}/dist/plugins", "{projectRoot}/build/plugins"],
+      "cache": true
     },
     "documentation": {
-      "outputs": [
-        "{workspaceRoot}/generated-doc/{projectName}/**/*"
-      ],
-      "inputs": [
-        "source"
-      ]
+      "outputs": ["{workspaceRoot}/generated-doc/{projectName}/**/*"],
+      "inputs": ["source"],
+      "cache": true
+    },
+    "prepare": {
+      "cache": true
+    },
+    "build-cjs": {
+      "cache": true
+    },
+    "build-esm2015": {
+      "cache": true
+    },
+    "build-esm2020": {
+      "cache": true
+    },
+    "build-react": {
+      "cache": true
+    },
+    "build-panel": {
+      "cache": true
+    },
+    "build-wrapper": {
+      "cache": true
+    },
+    "build-schematics": {
+      "cache": true
+    },
+    "build-extension": {
+      "cache": true
+    },
+    "build-components": {
+      "cache": true
+    },
+    "build-tools": {
+      "cache": true
+    },
+    "copy-assets": {
+      "cache": true
+    },
+    "generate-translations": {
+      "cache": true
+    },
+    "extract-translations": {
+      "cache": true
+    },
+    "extract-components": {
+      "cache": true
+    },
+    "extract-style": {
+      "cache": true
+    },
+    "extract-rules-engine": {
+      "cache": true
+    },
+    "build-swagger": {
+      "cache": true
+    },
+    "i18n": {
+      "cache": true
+    },
+    "@nx/jest:jest": {
+      "cache": true,
+      "inputs": ["default", "^default"],
+      "options": {
+        "passWithNoTests": true
```

Comment:
I think `"passWithNoTests": true` should be the exception, not the default value

Reply:
agreed